tests/foreman/ui/test_branding.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for branding
+
+:Requirement: Branding
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Branding
+
+:Assignee: akjha
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.config import settings
+
+
+def test_check_version_login_screen(default_sat):
+    """Check correct version on the login screen
+
+    :id: f48110ad-29b4-49b1-972a-a70075a05732
+
+    :Steps: Curl the login screen
+
+    :expectedresults: Correct version depending on the snap
+
+    :CaseLevel: System
+
+    :BZ: 1315849, 1367495, 1372436, 1502098, 1540710, 1582476, 1724738, 1959135, 2076979
+    """
+    result = default_sat.execute(
+        f'curl -k {default_sat.url}/users/login | '
+        f'grep -e "LoginPage" -e "{settings.server.version.release}"'
+    )
+    assert 'LoginPage' in result.stdout
+    assert str(settings.server.version.release) in result.stdout
+
+
+def test_beta_on_login_screen(default_sat):
+    """Check 'Beta' on the login screen
+
+    :id: f4448b74-8b09-4e00-99a9-47c212b53eb5
+
+    :Steps: Curl the login screen
+
+    :expectedresults: Should fail until a GA snap is received
+
+    :CaseLevel: System
+
+    :BZ: 1687250, 1686540, 1742872, 1805642,
+    """
+    result = default_sat.execute(f'curl -k {default_sat.url}/users/login | grep "Beta"')
+    assert result.status == 1, '"Beta" should not be there'
```

Comment:
I guess, you mean to assert this error when beta exists, and not if doesn't exist, right?

Reply:
This test should fail until we get a GA snap.
I will have a cherry-pick for 6.10 wherein this test should pass.